apollo-federation/src/sources/connect/json_selection/parser.rs

```diff
@@ -263,7 +263,7 @@ impl ExternalVarPaths for NamedSelection {
 
 #[derive(Debug, PartialEq, Eq, Clone)]
 pub struct PathSelection {
-    pub(super) path: WithRange<PathList>,
+    pub(crate) path: WithRange<PathList>,
```

Comment:
As far as I know, the only way to keep the `PathList` private in Rust is to avoid opening up its visibility like this. Please reconsider this change.